index.html

```diff
@@ -0,0 +1,235 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>African Movies Festival</title>
+    <link rel="stylesheet" href="./styles/mobile.css">
+    <link media="screen and (min-width: 768px)" rel="stylesheet" href="./styles/desktop.css">
+</head>
+
+<body>
+    <navbar class="navabar">
```

Comment:
- [ ] I would like to recommend you use to use The <nav> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element which it uses to represent a section of a page whose purpose is to provide navigation links, either within the current document or to other documents. Please fix it  for both about and index pages navigation:+1: